src/main/resources/org/openjdk/jextract/impl/resources/RuntimeHelper.java.template

```diff
@@ -24,6 +24,7 @@ import java.lang.foreign.MemoryLayout;
 
 import static java.lang.foreign.Linker.*;
 import static java.lang.foreign.ValueLayout.*;
+import static java.lang.Long.MAX_VALUE;
```

Comment:
In principle, this is not required. But one test was failing as it was declaring a struct called "Long" :-)